### PR TITLE
gsoler3 Acrolinx quality push: ad-forest-recovery-nonauthoritative-re…

### PR DESCRIPTION
**Pull request by gswashington**
…store.md

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
#assign: @robinharwood, @Xelu86 

@robinharwood @Xelu86 
Can you review the proposed changes? 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: WindowsServerDocs/identity/ad-ds/manage/AD-Forest-Recovery-Nonauthoritative-Restore.md [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
 
->Applies to: Windows Server 2022, Windows Server 2019, Windows Server 2016, Windows Server 2012 and 2012 R2, Windows Server 2008 and 2008 R2
+Applies to: Windows Server (All supported versions)
 
-To perform a nonauthoritative restore, complete the following procedure.
+The guidance in this article is intended to help you perform a nonauthoritative restore of Active Directory Domain Services.
 
-The following procedures use the Wbadmin.exe to perform a nonauthoritative restore of Active Directory or Active Directory Domain Services (AD DS). If you are using a different backup solution or if you intend to complete the authoritative restore of SYSVOL later in the forest recovery process, you can perform an authoritative restore of SYSVOL by using these alternative methods:
+The following procedures use the Wbadmin.exe to perform a nonauthoritative restore of Active Directory or Active Directory Domain Services (AD DS). If you're using a different backup solution or if you intend to complete the authoritative restore of SYSVOL later in the forest recovery process, you can perform an authoritative restore of SYSVOL by using these alternative methods:
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-ds/manage/AD-Forest-Recovery-Nonauthoritative-Restore.md:15]: Could you make this less complex?

> If you're using a different backup solution or if you intend to complete the authoritative restore of SYSVOL later in the forest recovery process, you can perform an authoritative restore of SYSVOL by using these alternative methods:

File: WindowsServerDocs/identity/ad-ds/manage/AD-Forest-Recovery-Nonauthoritative-Restore.md [hunk 2]
```diff
@@ -1,29 +1,27 @@
...
+- If you're using File Replication Service (FRS) to replicate SYSVOL, follow the steps in [article 290762](/troubleshoot/windows-server/networking/use-burflags-to-reinitialize-frs) in the Microsoft Knowledge Base, using the **BurFlags** registry key to reinitialize FRS replica sets. Alternate guidance in this [Knowledge Base article](https://support.microsoft.com/kb/315457) might also be helpful in rebuilding the SYSVOL tree. To determine if SYSVOL is replicated by FRS, see [Determining Whether a Domain Controller's SYSVOL Folder is Replicated by DFSR or FRS](/windows/win32/vss/backing-up-and-restoring-an-frs-replicated-sysvol-folder#determining_whether_a_domain_controller_s_sysvol_folder_is_replicated_by_dfsr_or_frs).
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-ds/manage/AD-Forest-Recovery-Nonauthoritative-Restore.md:17]: Could you replace the passive voice here?

>To determine if SYSVOL **is replicated by** FRS, see [Determining Whether a Domain Controller's SYSVOL Folder is Replicated by DFSR or FRS]

File: WindowsServerDocs/identity/ad-ds/manage/AD-Forest-Recovery-Nonauthoritative-Restore.md [hunk 3]
```diff
@@ -1,29 +1,27 @@
...
 - If you are using Distributed File System (DFS) Replication to replicate SYSVOL, see [Perform an authoritative synchronization of DFSR-replicated SYSVOL](AD-Forest-Recovery-Authoritative-Recovery-SYSVOL.md).
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-ds/manage/AD-Forest-Recovery-Nonauthoritative-Restore.md:18]:
```suggestion
- If you're using Distributed File System (DFS) Replication to replicate SYSVOL, see [Perform an authoritative synchronization of DFSR-replicated SYSVOL](AD-Forest-Recovery-Authoritative-Recovery-SYSVOL.md).

```

File: WindowsServerDocs/identity/ad-ds/manage/AD-Forest-Recovery-Nonauthoritative-Restore.md [hunk 4]
```diff
@@ -1,29 +1,27 @@
...
 
 ## Performing a nonauthoritative restore
 
-Use the following procedure to perform a nonauthoritative restore of AD DS and an authoritative restore of SYSVOL at the same time by using wbadmin.exe on a DC that runs Windows Server 2012, Windows Server 2008 R2, or Windows Server 2008. The backup must explicitly include system state data; a full server backup that is used for full server recovery will not work. For more information about creating a system state backup, see [Backing up the System State data](AD-Forest-Recovery-Backing-up-System-State.md).
+Use the following procedure to perform a nonauthoritative restore of AD DS and an authoritative restore of SYSVOL at the same time by using wbadmin.exe on a DC that runs Windows Server 2012, Windows Server 2008 R2, or Windows Server 2008. The backup must explicitly include system state data; a full server backup that's used for full server recovery won't work. For more information about creating a system state backup, see [Backing up the System State data](AD-Forest-Recovery-Backing-up-System-State.md).
```
Comment (robinharwood) [inline WindowsServerDocs/identity/ad-ds/manage/AD-Forest-Recovery-Nonauthoritative-Restore.md:22]: Could you make this less complex?

> Use the following procedure to perform a nonauthoritative restore of AD DS and an authoritative restore of SYSVOL at the same time by using wbadmin.exe on a DC that runs Windows Server 2012, Windows Server 2008 R2, or Windows Server 2008.

**Review by robinharwood — CHANGES_REQUESTED** [review]

**Comment by robinharwood** [comment]
Closing due to duplicated changes.

**Pull request closed without merging**